packages/localize/extension.package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@theia/localize",
+  "version": "0.1.1",
+  "description": "Theia - Localize Extension to translate string int different languages.",
+  "dependencies": {
+    "cldr-data": "^31.0.2",
+    "globalize": "^1.3.0",
+    "@types/globalize": "0.0.30",
+    "@types/mocha": "^2.2.41",
+    "chai": "^4.1.0",
+    "chai-as-promised": "^7.1.1",
```

Comment:
I don't think don't need to chia/mocha deps they are present in the package generator

Reply:
Ok, remove those dependencies